src/screens/OrganizationVenues/OrganizationVenues.spec.tsx

```
@@ -33,6 +33,13 @@
 import { vi } from 'vitest';
 import { errorHandler } from 'utils/errorHandler';
 
+// Mock MutationObserver
+global.MutationObserver = class {
+  constructor(callback: Function) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)